### ISSUE TITLE
Create PyPI package

### ISSUE THREAD

**Issue by jayvdb** [estelendur/dbfpy#1]
Hi, it would be great if you could create a PyPI package for this, to become a replacement for the old dbfpy.

Happy to help with packaging setup if you like.

**Comment by antmoth** [comment]
Certainly! I have no idea how to create a PyPI package, so I’d appreciate
your help. :)

If you or someone would be willing to review the couple of changes that I
made to handle the specific weirdnesses of the DBF files I was handling,
that would also be great. I’m not sure they are all generally applicable.

**Comment by jayvdb** [comment]
The first step is to clarify the licensing / attribution.

How much was derived from the previous 'dbfpy' package?

All commits are authored by you, so it isnt immediately obvious if you borrowed large chunks of code.
It doesnt matter if you did - but clarity is needed for distros to accept a package, so we may as well solve that now.

**Comment by jayvdb** [comment]
I see http://dbfpy.bzr.sourceforge.net/bzr/dbfpy/files has no mention of any license except public domain, which is a messy foundation, but it does make things easier as there are no terms to comply with.

**Comment by jayvdb** [comment]
https://packaging.python.org/ is *the* guide to follow.

If you go as far as creating a package on the test PyPI server, I will try using it as the source of an RPM package for openSUSE, to confirm it is ready to be published on the real PyPI server.

**Comment by jayvdb** [comment]
See https://github.com/kennethreitz/tablib/issues/273#issuecomment-471950390  , and potentially also look at working with @phargogh who produced https://github.com/phargogh/dbfpy3 , which has some open issues that may be also problems in this codebase.

**Comment by phargogh** [comment]
Hey everyone, happy to help here, both with packaging and with a python3 version of the package.  I'm sure there'll be some additional things to figure out, since work has continued on the upstream dbfpy since I ported v2.2.5 to python3.

**Comment by antmoth** [comment]
It looks like everything in the initial commit is from the sourceforge dbfpy, and everything after that is mine. Some of the changes are pretty haphazard because I was modifying it solely for internal use and based on the output of a proprietary program I don't have access to.

I will follow the abovelinked guide for creating a package.
